### PR TITLE
Fixes Blind staff dropping their stuff in arrivals

### PR DESCRIPTION
**Pull request by Esstoo**
[FIX][CLOTHING][TRAIT]

## About the PR 
People with Either the Blind, Short-Sighted or Deaf Trait combined with a Background Trait no longer spawn their stuff in arrivals at roundstart

## Why's this needed? 
Should Fix all of these:
#16758
#10323 
#12975 
#15270

### CODE REVIEW

**Review by ZeWaka — CHANGES_REQUESTED** [review]
yeah no, this code is super wack, and please indent your function bodies on a newline like i said

**Comment by Esstoo** [comment]
> alternatively, do the heavy sleeper thing and glue the old items onto the newly spawned person instead. Is it jank? yeah. Is it established precedent? Also yeah ([61b4859](https://github.com/goonstation/goonstation/commit/61b4859d288b0bbc4f264b668cba1df2886de028))

Makes more sense in my head if it stores it in your inventory.
If the backpack is full, it ends up in your hand, which will already glue the item to you anyway if you're a heavy sleeper.

File: code/mob/living/carbon/human.dm [hunk 1]
```diff
@@ -2283,6 +2283,19 @@
...
+		return
+	if (src.autoequip_slot(I, SLOT_L_STORE))
+		return
+	if (src.autoequip_slot(I, SLOT_R_STORE))
+		return
+	if (src.autoequip_slot(I, SLOT_L_HAND))
+		return
+	if (src.autoequip_slot(I, SLOT_R_HAND))
+		return
+	else qdel(I)
```
Comment (pgmzeta) [inline code/mob/living/carbon/human.dm:2297]: `else` would only trigger if the *last* `if` is false
```suggestion
	qdel(I)
```

File: code/mob/living/carbon/human.dm [hunk 2]
```diff
@@ -2283,6 +2283,19 @@
 	src.force_equip(I, slot)
 	return TRUE
 
+/mob/living/carbon/human/proc/stow_in_available(obj/item/I)
```
Comment (pgmzeta) [inline code/mob/living/carbon/human.dm:2286]: small doc-comment to explain functionality
```suggestion
///Tries to put an item in an available backpack, pocket, or hand slot; will delete the item if unable to place.
/mob/living/carbon/human/proc/stow_in_available(obj/item/I)
```
```

**Review by pgmzeta — CHANGES_REQUESTED** [review]
two small tweaks, looks good otherwise.

**Review by TobleroneSwordfish — APPROVED** [review]

**Pull request merged**